docs/learn-comparison-dfinity.md

```diff
@@ -0,0 +1,80 @@
+---
+id: learn-comparisons-dfinity
+title: Dfinity and Polkadot
+sidebar_label: Dfinity
+---
+
+Dfinity calls itself "the internet computer" and positions itself as a sister project to Ethereum. Dfinity has been
+in development since 2016, with a release called "Copper" supposed to come in 2017 but never seemed to have landed.
+In 2019, the world has yet to see the majority of Dfinity's closed source code for their client since they only publish
+helper libraries on their GitHub. The closed source nature of development is in stark contrast to the development of
+Polkadot, which has developed its Rust reference implementation in the open on GitHub from the start in 2017.
+
+### Algorithmic Governance
+
+Dfinity is a strong proponent of algorithmic governance, having published
+[two](https://medium.com/dfinity/the-dfinity-blockchain-nervous-system-a5dd1783288e#.duzxztt9k)
+[blog](https://medium.com/dfinity/future-governance-integrating-traditional-ai-technology-into-the-blockchain-nervous-system-825ababf9d9)
+posts explicitly on the topic. 
+
+Dfinity terms its governance framework the Blockchain Nervous System (BNS). The BNS awards a neuron token to 
+participants whom deposit Dfinity's native token in a system lock. The relative voting power of the neuron is weighted by the
+amounts of Dfinity tokens held in the deposit, and the delay for withdrawing these tokens is at least three months.
+The heaviest weighted neuron of the network then is able to dictate which direction the chain will progress when proposals
+would otherwise create blockchain forks. If the heaviest is not available, then it tries to follow the second heaviest,
+and so on until one neuron is available to decide the fork or none are (a situation not explored in their blog post).
+
+Dfinity positions its algorithmic governance as "AI is Law" in a similar vein to the Szabo-influenced "Code is Law" ethos.
+However, it makes this "Code is Law" statement contingent on the distributed intelligence of the blockchain nervous system.
+The reliance on external decision making to enact on-chain changes is comparable to Polkadot's approach to on-chain
+governance, in which the rules of participation are laid out in dry code but is ultimately dictated by the will of the
+token holders. However, the BNS seems to be a simplified reduction of governance to only deciding on chain forks through
```

Comment:
```suggestion
token holders. 

However, the BNS seems to be a simplified reduction of governance to only deciding on chain forks through
```

Reply:
Generally, paragraphs of over 3 sentences have been proven kill attention and make readers zone out.

Reply:
Nice, would be interested to see the source of this factoid. I instinctively agree with this but would like to see if there's a study.

Reply:
There were some writeups we did at SitePoint way back when: https://www.sitepoint.com/perfect-paragraph-length-even-care/, mostly based on other people's research and our own hotjar analytics on post content. I don't know of any actual studies though.